src/rextester.js

```diff
@@ -0,0 +1,137 @@
+        .post('https://rextester.com/rundotnet/api')
+        .set({
+            "Content-Type": "application/json"
+        })
+        .send({
+            LanguageChoice: language,
+            Program: code,
+            Input: "",
+            CompilerArgs: compilerArgs
+        })
+        .catch((err) => r = {
+            payload: err.toString(),
+            error: false,
+            code: 400
+        });
+    if (response.body.Result) {
+        if (response.body.Result.length == 0)
+            r = {
+                payload: 'Empty response',
+                error: false,
+                code: 200
+            }
+        else
+            r = {
+                payload: response.body.Result,
+                error: false,
+                code: 200
+            };
+
+    } else if (response.body.Errors) {
+        if (response.body.Errors.length == 0) r = {
+            payload: "Empty response (errored)",
+            error: true,
+            code: 200
+        }
+        else
+            r = {
+                payload: response.body.Errors,
+                error: true,
+                code: 200
+            };
+
+    }
+    return callback(r);
+}
+
+async function handleRequest(language, code, callback) {
+    let parseLangs = languageProperties.languageProperties.filter(i => i.classDeclaration != undefined).map(j => j.name); //The languages that have defined classes
+    let languageObject;
+    let imports = [];
+    let compilerArgs;
+
+    if (!code || !code[0]) return callback({
+        payload: 'no code/lang???',
+        error: false,
+        code: 400
+    });
+
+    code = code.split(' ');
+
+    languageProperties.languageProperties.forEach(curobject => { //Fetch the language that the user wants to use
+        if (curobject.aliases.includes(language.toString())) {
+            languageObject = curobject
+        }
+    });
+
+    if (!languageObject) return callback({
+        payload: 'Unsupported language',
+        error: false,
+        code: 400
+    });
+
+    if (parseLangs.includes(languageObject.name) && code[0] == languageObject.importType) { //If there are imports to parse then parse them
+        parseLanguages(code, imports)
+    };
+    if (code[0] == languageObject.importType && parseLangs.includes(languageObject.name)) { //Remove the imports after they have been parsed
+        code.splice(0, imports.length + 1)
+    }
+    code.join(" ")
+    if (languageObject.defaultImports.length > 0 || languageObject.classDeclaration) { //If there is a class to declare or if there are default imports then add them to code
+        code = await parseCode(code, languageObject, imports)
+    }
+    if(languageObject.languageCode) language = languageObject.languageCode
+    if(languageObject.compilerArgs) {
+        compilerArgs = languageObject.compilerArgs.join(" ")
+    } else {
+        compilerArgs = ""
+    }
+
+    outputResult(language, code, compilerArgs, async function (output) {
```

Comment:
There is no reason to make the callback async. Nor to `await callback()`.
This can also be an arrow function